requirements.txt

```diff
@@ -1,4 +1,4 @@
 click==8.1.7
-tox==4.15.0
-pytest==8.2.1
+tox==3.24.4
```

Comment:
was this commit made by mistake? Let's only adjust dependencies if we must.

Reply:
This change was implemented because of compatibility issues with the base image version from elifescience/sciencebeam-parser. It should be reverted to match the current versions in the main branch, as we are currently using a Debian image as the base.